label_studio/ml_models/models.py

```diff
@@ -110,6 +110,7 @@ class ModelRun(models.Model):
     class ProjectSubset(models.TextChoices):
         ALL = 'All', _('All')
         HASGT = 'HasGT', _('HasGT')
+        SAMPLE = 'Sample', _('Sample')
```

Comment:
Given that we plan to remove this field eventually and instead parameterize other subset types with something like `sample_n_tasks=`, what's the best migration plan for that in the future? Mark `Sample` as deprecated somehow or alias it to `All(sample_n_tasks=20)` without deleting it?

Reply:
Yeah, I'd say just deprecating it in the future is fine, personally

Reply:
I think we will still use this subset, and provide an additional subset size param, which I just added to the LSE PR

Reply:
this is implicitly a subset of `All`, but what if we wanted a subset of hasGT, or a subset of hasSubmitted or hasAccepted?